pkg/scm/git/git.go

```diff
@@ -217,83 +219,95 @@ func makePathAbsolute(source string) string {
 // expect git clone spec syntax; it also provides details if the file://
 // proto was explicitly specified, if we should use OS copy vs. the git
 // binary, and if a frag/ref has a bad format
-func ParseFile(source string) (details *FileProtoDetails, mods *URLMods) {
+func ParseFile(source string) (details *FileProtoDetails, mods *URLMods, err error) {
+	// NOTE: Why don't we just trim the source here?
```

Comment:
"these two things are basically the same so it doesn't really matter"

you're welcome to change it, but if you update fileExists to not strip file://, you need to make sure no one who is calling it expects it to retain that behavior.